### DISCUSSION TITLE
Ollama Upload Image to Model

### DISCUSSION THREAD

**Discussion by daniel156161** [danny-avila/LibreChat#2498] [Feature Requests & Suggestions]
### What happened?

Try to get a answer whats is in a Image with Ollama and the LLAVA model

### Steps to Reproduce

1. Docker-Compose install with Ollama
2. ollama pull llava
3. Custom Config with Ollama
4. Upload Image file into Ollama Chat

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

```shell
api Service:
2024-04-23 08:13:44 warn: [OpenAIClient.chatCompletion][stream] API error
2024-04-23 08:13:44 error: [handleAbortError] AI response error; aborting request: 400 json: cannot unmarshal array into Go struct field Message.messages.content of type string

ollama service:
[GIN] 2024/04/23 - 08:13:44 | 400 |     985.868µs |    192.168.32.7 | POST     "/v1/chat/completions"
```

### Screenshots

![image](https://github.com/danny-avila/LibreChat/assets/40464775/d34c1a21-f9ef-49d6-92dc-d2ba64761148)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
It's an Ollama limitation at the moment. Please let them know you would like to see this implemented: https://github.com/ollama/ollama/issues/3690

**Reply by djmaze** [reply]
It seems Ollama now supports this: https://github.com/ollama/ollama/pull/5208

**Comment by danny-avila** [comment]
I'm not sure Ollama will implement this, so I decided to add it:

![image](https://github.com/danny-avila/LibreChat/assets/110412045/ec55ad15-03ff-4664-b455-6a0a12dc996e)
